app/main.py

```diff
@@ -38,8 +39,22 @@ async def get_jobs(size: None | int = 500, active: None | str = 'true', ex: None
     url = settings.teraslice_url
 
     params = {'size': size, 'active': active, 'ex': ex}
-    # FIXME: figure out how to handle custom CA cert
-    r = httpx.get(f'{url}/jobs', params=params, verify=False)
+    
+    # Configure SSL verification - use custom CA cert if provided
+    verify_ssl = True
+    if settings.cacert_file:
+        verify_ssl = settings.cacert_file
+        logger.info(f"Using custom CA certificate: {settings.cacert_file}")
+    
+    try:
+        r = httpx.get(f'{url}/jobs', params=params, verify=verify_ssl)
```

Comment:
You treat verify_ssl as a boolean AND a string or file path here.  Does httpx.get REALLY accept a string as a value for verify?  Here's it's docs:

https://www.python-httpx.org/api/